autopush/utils.py

```diff
@@ -54,6 +58,65 @@ def generate_hash(key, payload):
     return h.hexdigest()
 
 
+def parse_header(header, major=";", minor="="):
+    """ Convert a multi-component header line (e.g. "a;b=c;d=e;...") to
+    a list.
+
+    For example, if the header content line is
+
+    `"a;c=1;b;d=2+3=5"`
```

Comment:
Oh, it's better than that. It could be called as 
a;c="fred";d=""george "smith"""

Fortunately, python makes handling that remarkably simple